src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -134,6 +134,8 @@
     {{ claimantWithDefendantAsBusinessResponse() }}
     {% elif status === ClaimStatus.CLAIMANT_ACCEPTED_DEFENDANT_PART_ADMISSION_AS_BUSINESS_WITH_ALTERNATIVE_PAYMENT_INTENTION_RESPONSE %}
     {{ claimantWithDefendantAsBusinessResponse() }}
+    {% elif ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_AS_BUSINESS_RESPONSE_WITH_MEDIATION %}
```

Comment:
this status doesn't exist any more please remove this if as the existing part admission code should now take care of it

Reply:
done